bot-components/GitHub_mutations.ml

```diff
@@ -32,6 +32,8 @@ let update_milestone ~bot_info ~issue ~milestone =
   | Error err ->
       Stdio.print_endline (f "Error while updating milestone: %s" err)
 
+type merge_method = [`MERGE | `REBASE | `SQUASH]
```

Comment:
I meant define a type with normal constructors (and convert internally those constructors into the required variants).